log4j-core/src/main/java/org/apache/logging/log4j/core/config/properties/PropertiesConfigurationBuilder.java

```diff
@@ -190,6 +193,17 @@ public PropertiesConfiguration build() {
         return builder.build(false);
     }
 
+    private static void useSyntheticLevelAndAppenderRefs(final String propertyName, final Properties loggerProps, final Properties context) {
+        final String propertyValue = (String) context.remove(propertyName);
+        if (propertyValue != null) {
+            final String[] parts = propertyValue.split(",");
```

Comment:
Do we need to trim each segment of `parts` to avoid whitespace issues? I suspect the level piece may handle it gracefully, but I'm less sure about the appenderRefs. Could be worth a test.

Reply:
Yes, that's a good point.

- [ ] Trim appender refs